public/js/index.js

```diff
@@ -0,0 +1,274 @@
+let subredditArray = [
+  [
+    "https://www.reddit.com/r/tifu.json",
+    "https://www.reddit.com/r/HighQualityGifs.json",
+    "https://www.reddit.com/r/science.json",
+    "https://www.reddit.com/r/gaming.json",
+    "https://www.reddit.com/r/Showerthoughts.json",
+    "https://www.reddit.com/r/MechanicalKeyboards.json",
+    "https://www.reddit.com/r/NoStupidQuestions.json",
+    "https://www.reddit.com/r/nonononoyes.json"
+  ],
+  "https://www.reddit.com/r/oddlysatisfying.json",
+  "https://www.reddit.com/r/catloaf.json",
+  "https://www.reddit.com/r/peoplefuckingdying.json"
+];
+
+//log down which subreddit was clicked
+let currentSubreddit;
+let redditJSONLimitAfter;
+//current date in seconds
+var date = new Date().getTime() / 1000;
+
+//navbar
+let findButtons = document.querySelectorAll(".subreddits");
+//Post area
+let postArea = document.getElementById("postArea");
+
+//Click navbar to load
+for (let i = 1; i < findButtons.length; i++) {
+  findButtons[i].addEventListener("click", grabData);
+  function grabData() {
+    postArea.innerHTML = "";
+    let subreddits = new XMLHttpRequest();
+    subreddits.open("GET", subredditArray[i]);
+    subreddits.send();
+    subreddits.addEventListener("load", loadData);
+  }
+}
```

Comment:
It's probably better to bind the eventListener to the buttons directly instead of using a for loop

Reply:
I was planning to make an add subreddit button that adds a new tab to the navbar and adds the .json link to my subredditArray so that I wouldn't need to manually make another eventListener for that new subreddit.

Reply:
ah okay understood.